content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

```diff
@@ -108,40 +108,61 @@ my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
 You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
 pod in this list.
 
+### Enable leader election
+
 To run multiple-scheduler with leader election enabled, you must do the following:
 
 First, update the following fields in your YAML file:
 
 * `--leader-elect=true`
-* `--lock-object-namespace=lock-object-namespace`
-* `--lock-object-name=lock-object-name`
+* `--lock-object-namespace=<lock-object-namespace>`
+* `--lock-object-name=<lock-object-name>`
+
+Note that the objects will be created, but the namespace must already exists.
+You can use the `kube-system` namespace.
 
-If RBAC is enabled on your cluster, you must update the `system:kube-scheduler` cluster role. Add your scheduler name to the resourceNames of the rule applied for endpoints resources, as in the following example:
+If RBAC is enabled on your cluster, you must update the `system:kube-scheduler` cluster role. Add your scheduler name to the resourceNames of the rule applied for `endpoints` and `leases` resources, as in the following example:
```

Comment:
How does this interact with EndpointSlice?

Reply:
kube-scheduler doesn't use them.